swank/src/main/scala/org/ensime/server/protocol/swank/SwankFormats.scala

```diff
@@ -442,6 +442,7 @@ object SwankProtocolResponse {
   }
   // must be after FileEditFormat
   implicit val RefactorEffectFormat = SexpFormat[RefactorEffect]
+  implicit val RefactorDiffEffectFormat = SexpFormat[RefactorDiffEffect]
```

Comment:
I'm impressed that you worked out the changes needed in this file :smile: it's really quite horrible. I'd love to just delete the whole thing, but it requires a big migration on the emacs side.